pkg/console/subresource/deployment/deployment.go

```diff
@@ -154,6 +171,27 @@ func Stub() *appsv1.Deployment {
 	return dep
 }
 
+func LogDeploymentAnnotationChanges(client appsclientv1.DeploymentsGetter, updated *appsv1.Deployment) {
```

Comment:
This is intended to ensure we can trace what object may have caused the deployment to roll pods.